docs/doc/03-develop/04-rust.md

```diff
@@ -60,7 +60,7 @@ use tokio_stream::StreamExt;
 
 #[tokio::main]
 async fn main() {
-    let dsn = "databend://user1:abc123@localhost:8000/default?sslmode=disable";
+    let dsn = "databend://user1:abc123@localhost:8000/book_db?sslmode=disable";
```

Comment:
We should change this since this database is not created yet.
We should change:
```
CREATE TABLE book_db.books
```